python/pyspark/sql/functions.py

```diff
@@ -426,6 +426,29 @@ def monotonically_increasing_id():
     return Column(sc._jvm.functions.monotonically_increasing_id())
 
 
+@since(2.1)
+def monotonically_increasing_id(offset):
```

Comment:
does it even work with python to do overloading this way? I think you are actually breaking things here.